Atomic/syscontainers.py

```diff
@@ -32,10 +32,12 @@
 
 ATOMIC_LIBEXEC = os.environ.get('ATOMIC_LIBEXEC', '/usr/libexec/atomic')
 ATOMIC_VAR = '/var/lib/containers/atomic'
-
+ATOMIC_VAR_USER = "%s/.containers/atomic" # Must be expanded
```

Comment:
in general I would prefer that code is not executed on an import, but this is small enough that I can fix in the patch and adjust the users of the variable